civis/resources/_resources.py

```diff
@@ -421,10 +438,24 @@ def parse_swagger(swagger, api_version, resources):
 
 
 @lru_cache(maxsize=4)
-def get_swagger_spec(api_key, user_agent, api_version):
+def get_swagger_spec(api_key, user_agent=None, api_version="1.0"):
+    """Download the Civis API specification.
+
+    Parameters
+    ----------
+    api_key : str
+        Your API key obtained from the Civis Platform. If not given, the
+        client will use the :envvar:`CIVIS_API_KEY` environment variable.
```

Comment:
At this point in the code, using the environment variable is no longer an option.